cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiTablePostgresSinkConsumerIT.java

```diff
@@ -0,0 +1,170 @@
+        ybHelper2 = new YBHelper(InetAddress.getLocalHost().getHostAddress(), TABLE_2);
+        pgHelper = new PgHelper(postgresContainer, TABLE_1);
+        pgHelper2 = new PgHelper(postgresContainer, TABLE_2);
+    }
+
+    @BeforeEach
+    public void beforeEachTest() throws Exception {
+        // Create table in the YugabyteDB database
+        ybHelper.execute(UtilStrings.getCreateTableYBStmt(TABLE_1, 10));
+
+        // Create another table with a composite primary key - id,first_name
+        String CREATE_STMT = "CREATE TABLE %s (id INT, first_name VARCHAR(30), last_name VARCHAR(50), days_worked DOUBLE PRECISION, PRIMARY KEY (id,first_name)) SPLIT INTO 10 TABLETS;";
+        ybHelper.execute(String.format(CREATE_STMT, TABLE_2));
+
+        // Initiate the cdcsdkContainer
+        // There will be total 20 tablets so the log line will come 20 times
+        cdcsdkContainer = kafkaHelper.getCdcsdkContainer(ybHelper, createTableIncludeListWithTableName(Arrays.asList(TABLE_1, TABLE_2)), 20);
+        cdcsdkContainer.withNetwork(containerNetwork);
+        cdcsdkContainer.start();
+
+        // Register the sink connectors
+        sinkConfig1 = pgHelper.getJdbcSinkConfiguration(postgresContainer, "id");
+        sinkConfig2 = pgHelper2.getJdbcSinkConfiguration(postgresContainer, "id,first_name");
+        kafkaConnectContainer.registerConnector(CONNECTOR_1, sinkConfig1);
+        kafkaConnectContainer.registerConnector(CONNECTOR_2, sinkConfig2);
+    }
+
+    @AfterEach
+    public void afterEachTest() throws Exception {
+        // Stop the cdcsdkContainer so that it doesn't cause any unexpected crashes
+        cdcsdkContainer.stop();
+
+        // Delete the sink connector from the Kafka Connect container
+        kafkaConnectContainer.deleteConnector(CONNECTOR_1);
+        kafkaConnectContainer.deleteConnector(CONNECTOR_2);
+
+        // Delete the topic in Kafka Container
+        kafkaHelper.deleteTopicInKafka(ybHelper.getKafkaTopicName());
+        kafkaHelper.deleteTopicInKafka(ybHelper2.getKafkaTopicName());
+
+        // TODO: Drop tables
+        dropTablesAfterEachTest(TABLE_1);
+        dropTablesAfterEachTest(TABLE_2);
+    }
+
+    @AfterAll
+    public static void afterAll() throws Exception {
+        // Stop the running containers
+        postgresContainer.stop();
+        kafkaConnectContainer.stop();
+        kafkaContainer.stop();
+    }
+
+    @Test
+    public void insertDataInBothSourceTablesAlternatively() throws Exception {
+        int recordsToBeInserted = 10;
+        for (int i = 1; i <= recordsToBeInserted; ++i) {
+            ybHelper.execute(UtilStrings.getInsertStmt(TABLE_1, i, "first_" + i, "last_" + i, 23.45));
+
+            int j = (-1) * i;
+            ybHelper2.execute(UtilStrings.getInsertStmt(TABLE_2, j, "first_" + j, "last_" + j, 123.45));
+        }
+
+        // Wait for records to be reflected across postgres
+        Thread.sleep(5000);
```

Comment:
Will you be making the change to use awaitility with `checkRecordCountInPostgres`?

Reply:
@Sumukh-Phalgaonkar will be raising a PR for the issue #31. My PR will only be merged after we get his PR merged.